### DISCUSSION TITLE
AIIdrive

### DISCUSSION THREAD

**Discussion by joluga1948** [BimmerGestalt/AAIdrive#762] [General]
Hola,por la app AIIAdrive,no puedo escuchar spotify, por Bluetooth ,en cambio por multimedia,si se escucha,sin carátulas,hace un tiempo me funcionaba,no se que habrá pasado,llevo el Idrive4++, gracias

**Comment by joluga1948** [comment]
A la espera dd soluciones,gracias

**Comment by hufman** [comment]
I don't quite follow the translation, I'm sorry.
IDrive4 does not support USB audio on modern Android phones, so make sure the IDrive is set to listen to Bluetooth just like if you were playing Spotify without AAIdrive.
Do the AAIdrive apps show up in the car? If not, you should figure out how to get MyBMW/Mini connected. They have bugs with USB support, so you may want to try Connected 6.4 or Connected Classic for better USB support,
